bokeh/models/tools.py

```diff
@@ -186,11 +186,12 @@ class TapTool(Tool):
 
 @abstract
 class InspectTool(Tool):
+    pass
 
-    active = Bool(True, help="""
-    Whether the tool is intially active or not. If set to ``False``, the user
-    will have to click tool's button to active it.
-    """)
+    #active = Bool(True, help="""
+    #Whether the tool is intially active or not. If set to ``False``, the user
+    #will have to click tool's button to active it.
+    #""")
```

Comment:
@bryevdv, I'm not sure what to do about this. I think this could be moved to `Tool`. It won't be bullet proof, due to know issues (need for an actual tool manager).